.github/workflows/test.yml

```diff
@@ -155,3 +155,47 @@ jobs:
         run: |
           gh pr close ${{ github.event.pull_request.number }} --delete-branch --comment "CI looks good, this dependency bump would not cause problems, hence closing this trial PR."
           gh pr edit ${{ github.event.pull_request.number }} --remove-label "autoclose"
+
+  approvals1:
+    if: github.event_name == 'pull_request' && github.event.pull_request.state == 'open' && github.event.pull_request.head.repo.full_name == github.repository && github.event.pull_request.base.ref == 'master' && github.event.pull_request.changed_files == 1
+    runs-on: ubuntu-latest
+    steps:
+      - uses: actions/checkout@v5
+        with:
+          ref: 'master'
+      - name: Hand out `Changelog.md` approval
+        env:
+          GH_TOKEN: ${{ github.token }}
+        run: |
+          gh pr diff ${{ github.event.pull_request.number }} | grep -E '^--- a/Changelog.md$' \
+          && gh pr review ${{ github.event.pull_request.number }} --approve --body 'A change to `Changelog.md`? I can handle that!'
+
+  approvals2:
```

Comment:
When all works, I know how to put these 3 together even if the conditions are slightly different.

Reply:
#5594 take care of this.